services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/proxy.py

```diff
@@ -5,8 +5,10 @@
 from ....core.settings import DynamicSidecarProxySettings, DynamicSidecarSettings
 from ....models.schemas.dynamic_services import SchedulerData, ServiceType
 
-MEMORY_50MB = 52430000
-CPU_1_PERCENT = 10000000
+MEMORY_50MB = 52_428_800
```

Comment:
These constants I find the useful. Why not moving them to ``packages/models-library/src/models_library/services_resources.py`` ??

Reply:
TIP: Use pydantic ByteSize. And it seems that MB now means power of 10. Mib means power of 2.
```parse_obj_as(ByteSize, "50Mib")``` is what you want